experimental/dds/tree2/src/shared-tree/sharedTreeView.ts

```diff
@@ -71,19 +70,9 @@ export interface ViewEvents {
 	 * A revertible change has been made to this view.
```

Comment:
I guess the second bullet here is temporary based on skipped test cases. fine to exclude that.

Reply:
I included the second cause it will still apply

Reply:
what's the plan of record for

		// TODO: unskip once forking revertibles is supported
		it.skip(`${name} (act on view undo on fork)`, () => {

?

Reply:
this will be done after mvp, the fork api will support passing in a list of revertibles that will get forked to work with the new fork and then merge and rebase will also emit new revertibles that came from other branches

Reply:
Cool, makes sense.